File: Algorithms/Fizz Buzz.js
// https://leetcode.com/problems/fizz-buzz/

/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
  var result = [], i

  for (i = 1; i <= n; i++) {
    if (i % 3 == 0) {
      
      if (i % 5 == 0) {
        result.push('FizzBuzz')
      } else {
        result.push('Fizz')  
      }

    } else {

      if (i % 5 == 0) {
        result.push('Buzz')
      } else {
        result.push(String(i))
      }

    }
  }
  
  return result
};

console.log(fizzBuzz(15))

File: 412-fizz-buzz/fizz_buzz.go
package solution

import (
	"strconv"
)

func fizzBuzz(n int) []string {
	ret := []string{}
	for i := 1; i <= n; i++ {
		switch {
		case i%15 == 0:
			ret = append(ret, "FizzBuzz")
		case i%3 == 0:
			ret = append(ret, "Fizz")
		case i%5 == 0:
			ret = append(ret, "Buzz")
		default:
			ret = append(ret, strconv.Itoa(i))
		}
	}
	return ret
}
